server/config/config.js

```diff
@@ -10,6 +10,7 @@ function config() {
   const env = process.env.NODE_ENV;
 
   return {
+    devtool: 'inline-eval-cheap-source-map',
```

Comment:
What does this do?

Reply:
oops, was trying out something but it didnt work out. fogot to remove